### PR TITLE
[feat] Connect teams data to back-end

### PR DESCRIPTION
**Pull request by hay92k**
Connect Teams data to back-end,
Filter matches for team

### CODE REVIEW

File: clients/app/src/stores/matches.ts [hunk 1]
```diff
@@ -27,14 +27,9 @@ const buildMatchesStore = (params: Partial<ParamType> = {}) => {
...
       return sdk.getMatches({
-        ...filter,
-        // dateFrom: '2022-04-05',
-        // dateTo: '2022-05-30',
-        // leagueId: 254,
-        // teamId: 2124,
         orderAttr: 'dateTime',
         order: OrderEnum.desc,
-        // limit: 3,
+        ...filter,
```
Comment (rubenaprikyan) [inline clients/app/src/stores/matches.ts:32]: moved lines to override order and orderAttrs as well ?

File: clients/app/src/views/screens/Team/components/TeamTabNavigation.tsx [hunk 2]
```diff
@@ -1,41 +1,39 @@
 import React from 'react';
 import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs';
 import { withTheme } from 'styled-components/native';
-
 import { teamTabScreens } from 'views/navigation/screens';
 import { ITeamTabNavigationProps } from 'views/navigation/types';
 
-const Stack = createMaterialTopTabNavigator();
-
+const TopTab = createMaterialTopTabNavigator();
```
[the remark this answers is not in the document]
Comment (hay92k) [reply clients/app/src/views/screens/Team/components/TeamTabNavigation.tsx:7]: yes, just the name changed.

File: clients/app/src/stores/matches.ts (hunk 1 above)
Comment (hay92k) [reply clients/app/src/stores/matches.ts:32]: yes.

File: clients/app/src/views/screens/Team/components/TeamTabNavigation.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (rubenaprikyan) [reply clients/app/src/views/screens/Team/components/TeamTabNavigation.tsx:7]: ok

File: clients/app/src/stores/matches.ts (hunk 1 above)
Comment (rubenaprikyan) [reply clients/app/src/stores/matches.ts:32]: well done

**Review by rubenaprikyan — APPROVED** [review]
lgtm!

**Pull request merged**